src/main/webapp/script.js

```diff
@@ -98,21 +99,20 @@ function loadHeatmap(data) {
     aqLayer.setMap(map);
 }
 
-function searchLocation() {
-    const locationString = document.getElementById("location-search-bar").value;
-
+function selectLocation(place) {
```

Comment:
It doesn't appear that any `select`ing is happening here, perhaps this would make more sense being called `markLocation`, since it appears to move the location of the marker to the place?

Reply:
Hmmmm, I feel like a verb similar to 'zoom in on' would be good, but I can't think of one. This function will eventually lead to the features for getting a route to that location and clean air near me as well. The marker was more to test that it was doing the right thing until those features happen

Reply:
Would `focusOnLocation` be an appropriate name?

Reply:
@sbland-g ?

Reply:
I've renamed it to setEndpoint as suggested